scipy/stats/_rvs_sampling.py

```diff
@@ -168,3 +171,451 @@ def rvs_ratio_uniforms(pdf, umax, vmin, vmax, size=1, c=0, random_state=None):
         i += 1
 
     return np.reshape(x, size1d)
+
+
+class NumericalInverseHermite:
+    r"""
+    A Hermite spline fast numerical inverse of a probability distribution.
+
+    The initializer of `NumericalInverseHermite` accepts `dist`, an object
+    representing a continuous distribution, and provides an object with methods
+    that approximate `dist.ppf` and `dist.rvs`. For most distributions,
+    these methods are faster than those of `dist` itself.
```

Comment:
after the change of the interface, dist will not have rvs in general

Reply:
True although it could be argue that it's a figure of speech here and we are talking about a general distribution and sampling from it. This could be rephrased maybe. IMO it's non blocking though. Or do you say it has to be changed?

Reply:
No, that is a minor point